### ISSUE TITLE
Small backpack upgraded to medium shows only 1 row until opened

### ISSUE THREAD

**Issue by ThatGravyboat** [BiscuitDevelopment/SkyblockAddons#156]
When you craft a medium backpack with an already opened small backpack the medium backpack hover GUI will show 1 row until opened. As shown in the screenshot below.
![2019-10-15_15 15 14](https://user-images.githubusercontent.com/26934102/66856073-036e4280-ef5f-11e9-90b4-b89c0eed1c08.png)

**Comment by biscuut** [comment]
Yeah this is because of the way Hypixel handles backpacks, so I don't think its anything we should really fix. The NBT of the backpack doesn't get changed until you open it, so the tag is still the small backpack one (which is why you also can't preview a fresh backpack without opening it, because the NBT tag doesn't exist yet).

**Comment by MCGamer00000** [comment]
> Yeah this is because of the way Hypixel handles backpacks, so I don't think its anything we should really fix. The NBT of the backpack doesn't get changed until you open it, so the tag is still the small backpack one (which is why you also can't preview a fresh backpack without opening it, because the NBT tag doesn't exist yet).

Here's an idea, its a bit out of the way, but it would work. If the tag says that its a small backpack, but it has the name of "Medium Backpack", then simply display the first 9 items as normal and then add the 9 more empty slots like there would be in a normal medium backpack. The same could happen for any other time the tag doesn't sync up with the name. Just add empty slots up to where the backpack name says it should be.

Then again though, it isn't really hurting anything and its a bit of a waste of time, so I can understand if you'd rather not do it.

**Comment by biscuut** [comment]
@MCGamer00000 sure i might as well make a workaround, i mean at the beginning i didnt want to because i wanted to display the data exactly as its stored, but i might as well since so many people think its a bug xd
